crates/transaction-emitter-lib/src/emitter/submission_worker.rs

```diff
@@ -62,21 +62,17 @@ impl SubmissionWorker {
             params,
             stats,
             txn_generator,
-            worker_index,
+            start_sleep_duration,
             skip_latency_stats,
             rng,
         }
     }
 
     #[allow(clippy::collapsible_if)]
     pub(crate) async fn run(mut self) -> Vec<LocalAccount> {
-        // Introduce a random jitter between, so that:
-        //  - we don't hammer the rest APIs all at once.
-        //  - allow for even spread for fixed TPS setup
-        let start_sleep_duration = self.start_sleep_time();
-        let start_time = Instant::now() + start_sleep_duration;
+        let start_time = Instant::now() + self.start_sleep_duration;
 
-        self.sleep_check_done(start_sleep_duration).await;
+        self.sleep_check_done(self.start_sleep_duration).await;
```

Comment:
I'm obviously missing something. How does this alter the wait time between requests, and not just the start wait time?

Reply:
it only affects the start wait time. 
It creates a pattern within the "wait_millis", that then repeats every "wait_millis" seconds